sdk/schemaregistry/azure-data-schemaregistry-avro/src/samples/java/com/azure/data/schemaregistry/avro/generatedtestsources/PlayingCard.java

```diff
@@ -71,9 +71,9 @@ public static PlayingCard fromByteBuffer(
     return DECODER.decode(b);
   }
```

Comment:
why we have changes on this file? do we need to revert this?

Reply:
I must have picked these up somewhere. I’ll revert then.